app/controllers/amusement_parks_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AmusementParksController < ApplicationController
+  def show
+    @amusement_park = AmusementPark.find(params[:id])
+    @mechanics = @amusement_park.rides.map(&:mechanics).flatten.uniq
```

Comment:
For the query used to retrieve a list of unique mechanics at the amusement park, you are currently using Ruby to process the data to retrieve this information over using AR and joins. 

According to the rubric, under the ActiveRecord category, in the row for Approaching Expectations, it states: "Student uses built-in ActiveRecord methods to process data, but may have one instance where they use Ruby instead"

Currently, you do have an instance where Ruby is being used over AR. A joins query was necessary to solve this problem, and that method should have been abstracted away into the Model.